### ISSUE TITLE
Doesn't respect `--target-binary-version` flag.

### ISSUE THREAD

**Issue by joshuapinter** [Microsoft/appcenter-cli#313]
We generate our `versionName` and `versionCode` values dynamically, so with `code-push` in the past we've had to manually specify the target binary version using the flag `--targetBinaryVersion`.

First off, with `appcenter` that flag is now listed as `--target-binary-version`. That's fine.

However, even if you specify the `--target-binary-version` or `-t` flag when using `appcenter codepush release-react`, it still complains about not being able to find the version and actually says you should use the `--targetBinaryVersion` flag: 

```bash
Detecting android app version:

Error: Command 'codepush release-react -a joshuapinter/CNTRAL-Mobile --deployment-name Staging --entry-file index.android.js --gradle-file ./cntral/build.gradle -t 6.53.0' failed with exception "The "android.defaultConfig.versionName" property value in "./cntral/build.gradle" is not a valid string. If this is expected, consider using the --targetBinaryVersion option to specify the value manually."
```

Neither of those flags work, tho.

**Comment by max-mironov** [comment]
@joshuapinter thanks so much for pointing at this, appreciate it! 
Here is PR that should fix it: https://github.com/Microsoft/appcenter-cli/pull/314

**Comment by christav** [comment]
Fixed in release 1.0.4.
